portal-frontend/src/app/features/applications/view-submission/lfng-review/lfng-review.component.html

```diff
@@ -21,6 +13,39 @@ <h2>Local/First Nation Gov Review</h2>
       >
         Download PDF
       </button>
+      <button mat-flat-button color="primary" *ngIf="application && application.canEdit" 
+        [routerLink]="'/application/' + application.fileNumber + '/edit'">
+        <div class="continue-button-content">
+          <span>Continue Application</span>
+          <mat-icon>keyboard_double_arrow_right</mat-icon>
+        </div>
+      </button>
+      <button (click)="onReview(application.fileNumber)"  mat-flat-button color="primary"
+        *ngIf="
+          application && application.canReview && (
+          application.status.code === SUBMISSION_STATUS.IN_REVIEW_BY_LG || 
+          application.status.code === SUBMISSION_STATUS.RETURNED_TO_LG || 
+          application.status.code === SUBMISSION_STATUS.SUBMITTED_TO_LG)
+        ">
+        <div class="continue-button-content">
+          <span
+            *ngIf="
+              application.status.code === SUBMISSION_STATUS.IN_REVIEW_BY_LG ||
+              application.status.code === SUBMISSION_STATUS.RETURNED_TO_LG
+            "
+            (click)="onReview(application.fileNumber)"
+            class="subheading2"
+            >Continue Application Review</span
+          >
+          <span
+            *ngIf="application.status.code === SUBMISSION_STATUS.SUBMITTED_TO_LG"
+            (click)="onReview(application.fileNumber)"
+            class="subheading2"
+            >Start Application Review</span
+          >
```

Comment:
Code improvement: There's a lot of repetition of the same onClick function here. I can see that the code was moved from another file, but would it be possible to remove the ones from the conditional <span>, and use the enclosing one from <button>?

Reply:
we have a refactoring ticket which was moved to another sprint. i am focusing on solving this now. and we can refactor in other one.

Reply:
https://apps.nrs.gov.bc.ca/int/jira/browse/ALCS-1403